src/commands/CommandMusicPlay.java

```diff
@@ -138,7 +166,8 @@ public String getCommandDescription(){
 	public ParametersHelp[] getParametersDescriptions(){
 		return new ParametersHelp[]
 		{
-			new ParametersHelp(lang("ReplayDescription"), "l", "latest")
+			new ParametersHelp(lang("ReplayDescription"), "l", "latest"),
```

Comment:
This ParameterHelp (and the one for `anything`/`random`) could use the new https://github.com/Vhoyon/Discord-Bot/blob/6b3a3714451ddd1a86d017b3619ea511519b49c7/framework/vendor/objects/ParametersHelp.java#L16-L19
constructor to prevent parameters from actually capturing content when they are only meant to be flags (this boolean, when sets to `false`, treats content after it as command's content).

Might as well use it for both from the beginning, other commands will also need some updating!